src/LEGO.AsyncAPI.Bindings/Sns/FilterPolicy.cs

```diff
@@ -0,0 +1,26 @@
+namespace LEGO.AsyncAPI.Bindings.Sns
+{
+    using System;
+    using LEGO.AsyncAPI.Models.Interfaces;
+    using LEGO.AsyncAPI.Writers;
+
+    public class FilterPolicy : IAsyncApiElement
+    {
+        /// <summary>
+        /// A map of a message attribute to an array of possible matches. The match may be a simple string for an exact match, but it may also be an object that represents a constraint and values for that constraint.
+        /// </summary>
+        public IAsyncApiAny Attributes { get; set; }
```

Comment:
I don't think 'Any' is s good fit here. But I'll check the specification. 

Any type, is already a wonky solution

Reply:
I've put a question in the sns binding PR to get clarification.

But if it's how it is, in the binding it should be Any type, of string, array or object. Which could be solved with a validation rule.

ReadAny exists but is probably internal in the readers project.